---
# Documentation: https://sourcethemes.com/academic/docs/managing-content/

title: Highly multiplexed spatially resolved gene expression profiling of mouse organogenesis
subtitle: ''
summary: ''
authors:
- T. Lohoff
- S. Ghazanfar
- A. Missarova
- N. Koulena
- N. Pierson
- J. A. Griffiths
- E. S. Bardot
- C.-H. L. Eng
- R. C. V. Tyser
- R. Argelaguet
- C. Guibentif
- S. Srinivas
- J. Briscoe
- B. D. Simons
- A.-K. Hadjantonakis
- B. GÃ¶ttgens
- W. Reik
- J. Nichols
- L. Cai
- J. C. Marioni
tags: []
categories: []
date: '2020-11-01'
lastmod: 2020-12-29T11:18:31Z
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2020-12-29T11:18:30.451431Z'
publication_types:
- '2'
abstract: textlessptextgreaterTranscriptional and epigenetic profiling of single-cells
  has advanced our knowledge of the molecular bases of gastrulation and early organogenesis.
  However, current approaches rely on dissociating cells from tissues, thereby losing
  the crucial spatial context that is necessary for understanding cell and tissue
  interactions during development. Here, we apply an image-based single-cell transcriptomics
  method, seqFISH, to simultaneously and precisely detect mRNA molecules for 387 selected
  target genes in 8-12 somite stage mouse embryo tissue sections. By integrating spatial
  context and highly multiplexed transcriptional measurements with two single-cell
  transcriptome atlases we accurately characterize cell types across the embryo and
  demonstrate how spatially-resolved expression of genes not profiled by seqFISH can
  be imputed. We use this high-resolution spatial map to characterize fundamental
  steps in the patterning of the midbrain-hindbrain boundary and the developing gut
  tube. Our spatial atlas uncovers axes of resolution that are not apparent from single-cell
  RNA sequencing data - for example, in the gut tube we observe early dorsal-ventral
  separation of esophageal and tracheal progenitor populations. In sum, by computationally
  integrating high-resolution spatially-resolved gene expression maps with single-cell
  genomics data, we provide a powerful new approach for studying how and when cell
  fate decisions are made during early mammalian development.textless/ptextgreater
publication: '*bioRxiv*'
url_pdf: https://www.biorxiv.org/content/10.1101/2020.11.20.391896v1
doi: 10.1101/2020.11.20.391896
---
